duckdb_engine/__init__.py

```diff
@@ -601,6 +603,30 @@ def get_multi_columns(
 
         return columns.items()
 
+    def _comment_query(self, schema, has_filter_names, scope, kind):
+        relkinds = self._kind_to_relkinds(kind)
```

Comment:
I would add a comment here saying why we need this, eg
`# fix for https://github.com/Mause/duckdb_engine/issues/1128`

Reply:
Done in the latest commit!